src/scraper/Classes/CurlHandler.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace TopDog\scraper\Classes;
```

Comment:
scraper should be capital S (directory name should be too)

Reply:
Fred